source/_integrations/neurio_energy.markdown

```diff
@@ -43,3 +43,75 @@ sensor_id:
   required: true
   type: string
 {% endconfiguration %}
+
+## Neurio hardware has an open local network accessible API
+
+The current implementation of the Neurio Energy component is based on a cloud based service, should that service end in the future, it is still possible to read the Neurio's Energy values for each CT and their consumption directly from the neurio hardware itself by hitting the device's IP with `/current-sample` at the end, i.e. `http://NEURIO-IP/current-sample`
```

Comment:
What has this to do with this integration?

Reply:
It has nothing to do with the current integration,  don't know if there's a place to document available APIs for a product associated with an integration. I could redo it as an alert, but I don't know that its appropriate for that either...

As mentioned in the commit comment, feel free to discard/reject it.

Reply:
> It has nothing to do with the current integration

In that case it doesn't belong here.

>  don't know if there's a place to document available APIs for a product associated with an integration

We don't have a place for that in our documentation. Feel free to list it in our community guides section of the community forum 👍